contents/handbook/strategy/strategy.md

```diff
@@ -5,41 +5,85 @@ showTitle: true
 ---
 
 ## TL;DR
-Our mission is to “Increase the number of successful products in the world”.
+Our mission is to increase the number of successful products in the world.
 
-We started by building an open-source product analytics platform with all the basic things you'd need to understand user behavior - funnels, trends, session recording, feature flags and many more features. Within a year we had thousands of customers using us and we started generating revenue.
+We started by building an open-source product analytics platform with all the basic things you'd need to understand user behavior - funnels, trends, session recording, feature flags, and many more features. Within a year, we had thousands of customers using us and we started generating revenue.
 
-We focused on our paid product earlier this year. We quickly hit a milestone for the first 5 reference customers, and we've kept selling since. The next step on our journey is to figure out how to accelerate our top of funnel growth, whilst achieving larger order values from our Enterprise product.
+We focused on our paid product earlier in 2021. We quickly hit a milestone for the first 5 reference customers and we've kept selling since. The next step on our journey is to figure out how to accelerate our top of funnel growth, while achieving larger order values from our Enterprise product.
 
-### Context
+## Context
 
-We’ve grown a lot (10s of thousands of sign ups), but it’s clear that the many of our users use us *despite* a bad experience - complex deployment and maintenance, in particular. This is a great problem to have. It means we’re solving a hair on fire problem - product analytics that you can self host.
+We’ve grown a lot - tens of thousands of sign ups - but it’s clear that the many of our users use us *despite* a bad experience, with complex deployment and maintenance a particular issue. This is a great problem to have, as it means we’re solving a hair-on-fire problem - product analytics that you can self-host.
 
 For any company, nothing matters more than product market fit. If we get that right, it’ll be much easier for customer success, marketing or sales teams to succeed.
 
-### Mission
+## Mission
 
 **_“Increase the number of successful products in the world”_**
 
-### Long-term vision (for 2023)
+## Long-term vision (for 2026)
 
-**_“Everyone building a product has a clear path to making it successful without losing control of their data”_**
+**Where do we want to get to?**
 
-This means engineers can install PostHog and within a few hours, qualitative and quantitative data will be captured from their product, audience and any other data sources they have (playing nicely with data warehouses).
+In 2026 we will _go public with $100M ARR._ To achieve this, Posthog will need to be _the standard devtool for building better products._
```

Comment:
Devtool makes it sound like it's not a product tool. We're targeting PMs too as key users.